MovieList/MovieList/MovieList/MovieListCoordinator.swift

```diff
@@ -57,6 +57,10 @@ extension MovieListCoordinator: MovieListVCDelegate {
         movieController.navigationController?.pushViewController(movieDetailVC, animated: true)
     }
     
+    func returnToSearchVC() {
+        movieController.navigationController?.popToRootViewController(animated: true)
```

Comment:
Whitespace here.

Reply:
I've checked the project and there's no whitespace in returnToSearchVC(). Maybe a GitHub thing?